apps/projex/src/app/app.component.html

```diff
@@ -1,66 +1,32 @@
 <biosimulations-navigation
-  [appName]="config.appName"
-  [appNameParts]="config.appNameParts"
-  [logo]="config.logo"
-  [privacyPolicyVersion]="config.privacyPolicyVersion"
-  [newsVersion]="1"
-  [showNews]="false"
-  [healthy]="healthy$ | async">
-  <biosimulations-topbar-menu>
-    <biosimulations-hover-open-menu [href]="config.platformAppUrl" [disabled]="true">
-      <biosimulations-topbar-menu-item heading="BioSimDB" icon="browse" id="trigger"></biosimulations-topbar-menu-item>
-    </biosimulations-hover-open-menu>
-
-    <biosimulations-hover-open-menu [href]="config.dispatchAppRunsNewUrl" [disabled]="true">
-      <biosimulations-topbar-menu-item heading="Run Simulations" icon="experiment" id="trigger">
-      </biosimulations-topbar-menu-item>
-    </biosimulations-hover-open-menu>
-
-    <biosimulations-hover-open-menu>
-      <biosimulations-topbar-menu-item heading="Utilities" icon="construction" id="trigger">
-      </biosimulations-topbar-menu-item>
-      <ng-container id="items">
-        <biosimulations-dropdown-menu-item heading="Convert a file" [href]="config.utilsConvertFileUrl">
-        </biosimulations-dropdown-menu-item>
-
-        <biosimulations-dropdown-menu-item heading="Create a project" [href]="config.utilsCreateProjectUrl">
-        </biosimulations-dropdown-menu-item>
-
-        <biosimulations-dropdown-menu-item heading="Validate a model" [href]="config.utilsValidateModelUrl">
-        </biosimulations-dropdown-menu-item>
-
-        <biosimulations-dropdown-menu-item heading="Validate a simulation" [href]="config.utilsValidateSimulationUrl">
-        </biosimulations-dropdown-menu-item>
-
-        <biosimulations-dropdown-menu-item heading="Validate metadata" [href]="config.utilsValidateMetadataUrl">
-        </biosimulations-dropdown-menu-item>
-
-        <biosimulations-dropdown-menu-item heading="Validate a project" [href]="config.utilsValidateProjectUrl">
-        </biosimulations-dropdown-menu-item>
-
-        <biosimulations-dropdown-menu-item
-          heading="Validate a simulator"
-          href="https://github.com/biosimulators/Biosimulators/issues/new?assignees=&labels=Validate%2Fsubmit+simulator&template=ValidateOrSubmitASimulator.yml&title=%5BSimulation+capabilities%5D%3A+"
-          target="githubissues">
-        </biosimulations-dropdown-menu-item>
-
-        <biosimulations-dropdown-menu-item heading="Suggest a simulator" [href]="config.utilsSuggestSimulatorUrl">
-        </biosimulations-dropdown-menu-item>
-
-        <biosimulations-dropdown-menu-item heading="BioSimulations REST API" [href]="config.apiUrl">
-        </biosimulations-dropdown-menu-item>
-      </ng-container>
-    </biosimulations-hover-open-menu>
-
```

Comment:
I know that /app/projex is going away soon, but here again the navigation for utilities is removed.  In this case, you could even hard code the *.org (or keep it in the config.json).